flexget/plugins/filter/translations.py

```diff
@@ -0,0 +1,594 @@
+import re
+from typing import List, Union
+from unicodedata import normalize
+
+import babelfish
+from babelfish.language import LANGUAGES
+from guessit.api import GuessItApi, GuessitException
+from loguru import logger
+
+from flexget import plugin
+from flexget.config_schema import one_or_more
+from flexget.event import event
+
+PLUGIN_NAME = 'translations'
+logger = logger.bind(name=PLUGIN_NAME)
+
+UNKNOWN = 'unknown'  # It's a language but is unknown
+DEFAULT = 'default'  # Default when none applies
+NATIVE = 'native'  # Native language
+NONE = 'none'  # No language
+OTHER = 'other'  # It's a valid language, but not matched
+ACTION_ACCEPT = 'accept'  # Accept entry
+ACTION_REJECT = 'reject'  # Reject entry
+ACTION_DO_NOTHING = 'do_nothing'  # Ignore check (internal)
+
+Language = babelfish.Language
+
+
+class MyCodeConverter(babelfish.LanguageEquivalenceConverter):
+    """
+    This Class will allow to declare all the match that babelfish is not able to match
+    """
+
+    SYMBOLS = {}
+
+
+babelfish.language_converters['mycode'] = MyCodeConverter()
+
+
+class Translations:
+    """
+    Take action on translated content
+
+    Example:
+
+    translations:
+      action: reject
+
+    translations:
+      languages:
+        - "imdb_languages"
```

Comment:
I don't quite follow, in some examples the "languages" key contains list of languages like "japanese" but here we have entry field names. If it has some intelligence/magic that allows both, I would much prefer to be explicit and have "language_fields" configuration key instead.

Reply:
I've split the fields, now we have languages and language_fields